docs/contributing/coding-guidelines/index.md

```diff
@@ -5,3 +5,12 @@
     Under Construction
 
 ## Common guidelines
+
+Refer to the following links for now:
+
+- <https://docs.ros.org/en/foxy/Contributing/Developer-Guide.html>
+
+Also, keep in mind the following concepts.
+
+- Make things consistent.
+- Don't stick to the parts that can't be automated.
```

Comment:
What is meant by `parts`? I think some more detail is required here to clarify what should be automated where possible.

Reply:
I think `things` or `items` is better? :thinking: 

> to clarify what should be automated where possible.

Hmm? Could you explain this a bit more in detail? :pray:

Reply:
Apologies for not getting back to you about this question. The original sentence was:

> Don't stick to the parts that can't be automated

In other words, we should automate whenever possible? If that is true, then I think that some more detail (examples?) is needed to make this concept more clear.

Reply:
Agreed. Like this? `Automate simple checks such as formatting, syntax, etc.`
Refeference: https://abseil.io/resources/swe-book/html/ch09.html#automate_where_possible

Reply:
```suggestion
- Automate where possible, such as simple checks for formatting, syntax, etc.
```

I think this would work.

Reply:
Thank you for your feedback.

The reason why I wrote the original sentence is to avoid small battles that come from people's preferences.
It's a kind of https://en.wikipedia.org/wiki/Law_of_triviality. (I should have linked this page earlier.)

I think this is a good page. Please let me know if you know some better pages. :pray: 
https://steemit.com/programming/@emrebeyler/code-reviews-and-parkinson-s-law-of-triviality

Reply:
```suggestion
- Automate where possible, using simple checks for formatting, syntax, etc.
- When it comes to code reviews, don't spend too much time on trivial disagreements. For details see:
  - https://en.wikipedia.org/wiki/Law_of_triviality
  - https://steemit.com/programming/@emrebeyler/code-reviews-and-parkinson-s-law-of-triviality
```

I see, what about this?

Reply:
Great! Thank you! :pray:

Reply:
I've added `<x>` but not sure if I should add `<x/>` for links.

Reply:
Oh pre-commit bot added `<>` it's ok.